angular/projects/public-nrpti/src/app/records/records-list/records-list.component.scss

```diff
@@ -1,177 +1,177 @@
-
-  &:focus {
-    box-shadow: 0 0 0 0.2rem rgba(0, 123, 255, 0.25);
-    border-color: #5091cd;
-  }
-}
+    margin-bottom: 1rem;
 
-.btn-search-help {
-  padding: 6px 12px;
-  margin-bottom: 0.5rem;
-  margin-right: 0.5rem;
-  transition: all ease-out 0.2s;
-  cursor: pointer;
-
-  .material-icons {
-    font-size: 20px;
-    margin-top: -3px;
-    margin-right: 2px;
-  }
-}
+    .inline-header {
+      > * {
+        display: inline-block;
+      }
 
-.btn-num-items {
-  margin-left: 0.5rem;
-}
+      i {
+        font-size: 1.25rem;
+        vertical-align: text-bottom;
+      }
+    }
+
+    .keyword-search {
+      @include flex-box();
+      @include flex(0 0 auto);
+      @include flex-direction(row);
+
+      height: 2.35rem;
+      margin-bottom: 1rem;
+      border: 2px solid $primary;
+
+      input {
+        @include flex(1 1 auto);
 
-.btn-outline-secondary {
-  color: #007bff;
-  border-color: #007bff;
+        padding: 0 0.5rem;
+        border: 0;
+      }
 
-  i {
-    margin-right: 4px;
-    font-size: 20px;
+      button {
+        @include flex(0 0 auto);
+
+        border: 0;
+        background-color: $primary;
+
+        i {
+          vertical-align: middle;
+          color: white;
+        }
+      }
+    }
+
+    button {
+      float: right;
+      background-color: $primary;
+    }
   }
 
-  &:disabled {
-    border-color: #6c757d;
-    color: #6c757d;
+  &__content {
+    border: 1px solid #036;
+    border-radius: 6px;
+    padding-left: 1rem;
+    padding-right: 1rem;
   }
 }
 
-.search-area {
-  background-color: white;
-  border: 1px solid lightgray;
+.grey-subtext {
+  color: $gray6;
+  margin-left: 0.5rem;
 }
 
-.main-search-field {
-  margin-left: 0;
-  padding: 8px;
-}
+.content-section {
+  .content-grid {
+    display: grid;
+    grid-gap: 1rem;
+    grid-template-rows: repeat(auto-fit, minmax(0, min-content));
+    grid-template-columns: 1fr 3fr;
+
+    &.show-advanced-filters {
+      grid-template-areas:
+        "Search  ListHeader"
+        "Search  ListContent";
+    }
 
-.show-advanced-search-button .btn {
-  float: right;
-  margin-right: 8px !important;
-  width: 200px;
-}
+    &:not(.show-advanced-filters) {
+      grid-template-areas:
+        "Search       ListHeader"
+        "ListContent  ListContent";
+    }
+
+    .left-column {
+      grid-area: Search;
+    }
 
-.advanced-search {
-  position: relative;
+    .right-column {
+      &__header {
+        grid-area: ListHeader;
+      }
 
-  .project-filter {
-    margin-left: 15px;
-    margin-top: 10px;
-    margin-right: 93px;
-    margin-bottom: 10px;
-  }
+      &__content {
+        grid-area: ListContent;
+      }
+    }
 
-  .filter-border {
-    border: solid 1px lightgray;
-    padding: 15px;
-    min-width: 155px;
-  }
+    // Can also be nested inside <grid-item-value> - font-size scales automatically (re: 0.8em).
+    // .grid-item {
+    //   display: grid;
+    //   grid-gap: 0.25rem;
+    //   font-size: 0.8em;
```

Comment:
Remove all references of commented out CSS

Reply:
removed